databus-client2/pom.xml

```diff
@@ -0,0 +1,102 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <parent>
+        <groupId>com.bazaarvoice.emodb</groupId>
+        <artifactId>emodb-parent</artifactId>
+        <version>6.5.4-SNAPSHOT</version>
+        <relativePath>../parent/pom.xml</relativePath>
+    </parent>
+
+    <artifactId>emodb-databus-client2</artifactId>
+    <packaging>jar</packaging>
+
+    <name>EmoDB Databus HTTP Jersey2 Client</name>
+
+    <dependencies>
+        <dependency>
+            <groupId>com.bazaarvoice.emodb</groupId>
+            <artifactId>emodb-databus-api</artifactId>
+            <version>${project.version}</version>
+        </dependency>
+        <dependency>
+            <groupId>com.bazaarvoice.emodb</groupId>
+            <artifactId>emodb-common-client</artifactId>
```

Comment:
this dependency should be removed to not include transitive jax-rs 1.x dependencies.
also please check `maven dependency:tree` for this module

Reply:
separating the dependency requires changes in the client module of Jersey 2 BlobStore, moving the effort of the changes to the ticket [PD-174028 ](https://bazaarvoice.atlassian.net/browse/PD-174028) since BlobStore changes are not in scope.

Reply:
seems like this PR depends on https://github.com/bazaarvoice/emodb/pull/404